src/views/quotas/list/components/AdditionalQuotaPopover/AdditionalQuotaPopover.scss

```
@@ -0,0 +1,4 @@
+.additional-quota-popover__description-list {
+  --pf-v6-c-description-list--RowGap: var(--pf-t--global--spacer--sm);
+  margin-top: var(--pf-t--global--spacer--md);
```

Stylelint (17.2.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)